lib/Conversion/TorchToStablehlo/Basic.cpp

```diff
@@ -432,7 +432,8 @@ class ConvertAtenMulDivOp : public OpConversionPattern<AtenOpT> {
     Value result =
         rewriter.create<ChloOpT>(loc, outType, lhs, rhs, bcastDimensions);
 
-    if (!isa<AtenDivTensorModeOp, AtenDivScalarModeOp>(&op)) {
+    if (!std::is_same<AtenDivTensorModeOp, AtenOpT>() ||
```

Comment:
Just changed isa to `is_same` since the type can be evaluated against tensor/scalar mode at compile time (if could be marked `constexpr` but idk how important that is)